lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -3635,21 +3811,22 @@ namespace Js
 
     void FunctionBody::SetStackNestedFuncParent(FunctionBody * parentFunctionBody)
     {
-        Assert(this->stackNestedFuncParent == nullptr);
+        Assert(this->get_AuxPtr(e_stackNestedFuncParent) == nullptr);
```

Comment:
I don't see this as a good reason to stray from our coding conventions.  See my comments below.